python/lsst/meas/astrom/pessimistic_pattern_matcher_b_3D.py

```diff
@@ -273,7 +273,7 @@ def match(self, source_array, n_check, n_match, n_agree,
         # Initialize output struct.
         output_match_struct = pipeBase.Struct(
             match_ids=[],
-            distances=[],
+            distances_rad=[],
```

Comment:
Also update the numpy docs above to match: the output is `distances_rad` instead of `distances`